Source/Scene/ViewportQuad.js

```diff
@@ -135,6 +135,7 @@ ViewportQuad.prototype.update = function (frameState) {
 
     this._material.update(context);
 
+    this._overlayCommand.renderState = this._rs;
```

Comment:
Thanks for the fix, @bbbbx! 

@lilleyse - This change makes sense to me but I'm unable to reproduce the bugged behavior on my Linux machine. Any thoughts on this fix? I wonder if it's somehow possible to simulate a render state change in the `ViewportQuad` specs.

Reply:
Should be straightforward to test by setting the rectangle after it's been created an observing that the command's render state has the correct viewport. Something like:

```
    it("updates rectangle", function () {
      var otherRectangle = new BoundingRectangle(0, 0, 4, 4);

      scene.primitives.add(viewportQuad);
      scene.renderForSpecs();

      viewportQuad.rectangle = otherRectangle;
      scene.renderForSpecs();
      expect(scene.frameState.commandList[0].renderState.viewport).toEqual(
        otherRectangle
      );
    });
```

Reply:
Also I can confirm this PR fixes the bug.

Reply:
Hi, I have added the test, could you review it again? Thanks @lilleyse @ebogo1